spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,40 @@
+# spec/integration/user_index_spec.rb
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :system, js: true do
+  before(:all) do
+    @john = User.create(name: 'John Cena', photo: 'https://picsum.photos/200/300',
+                        bio: 'Welcome to the new era of the WWE. The champ is here!')
+    @cristiano = User.create(name: 'Cristiano Ronaldo', photo: '/assets/default_user.jpg',
+                             bio: 'I am ubleivable inside the pitch.')
+
+    @post1 = Post.create(title: 'I am the champ', text: 'I am the champ of the WWE.', author: @john)
+    @post2 = Post.create(title: 'You can\'t see me', text: 'My time is now. It\'s the frenchise.', author: @john)
+    @post3 = Post.create(title: 'I will never give up', text: 'I will never give up. I will never surrender.',
+                         author: @john)
+    @post4 = Post.create(title: 'No one can beat me', text: 'No one can beat me. I am the best.', author: @john)
+    @post5 = Post.create(title: 'Like it or not', text: 'Like it or not, I am the best.', author: @john)
+  end
+
+  describe 'index page' do
+    before(:example) do
+      visit users_path
+    end
+
+    it 'should render user information' do
+      expect(page).to have_content(@cristiano.name)
+      expect(page).to have_content(@john.name)
+
+      expect(page).to have_css("img[src*='https://picsum.photos/200/300']")
+      expect(page).to have_css("img[src*='default_user.jpg']")
+
+      expect(page).to have_content(@cristiano.posts_counter)
+      expect(page).to have_content(@john.posts_counter)
```

Comment:
- [ ] @Team great work adding tests for the user index page, however, the tests that assert you can see the username,  and profile picture are supposed to check that each user profile picture and username is visible, kindly consider using each loop in your tests as you did in posts show page to achieve the objective.
- [ ] @Team you also forgot to add a test that assets ` you can see the number of posts the user has written.`
![image](https://github.com/serengia/rails-blog-app/assets/61408860/2b71885f-df91-4ad1-b805-41ef68a5a53d)